python/meeting-scheduling/src/meeting_scheduling/demo_data.py

```diff
@@ -0,0 +1,214 @@
+        "Fast track multitasking", "Cross sell one stop shop", "Reinvigorate multitasking"
+    ]
+    
+    meetings = []
+    for i, topic in enumerate(meeting_topics):
+        meeting = Meeting(id=str(i), topic=topic, duration_in_grains=0)
+        meetings.append(meeting)
+    
+    # Set durations using CountDistribution and random.choices
+    duration_distribution = (
+        CountDistribution(count=8, weight=1),   # 33% with 8 time grains
+        CountDistribution(count=12, weight=1),  # 33% with 12 time grains
+        CountDistribution(count=16, weight=1)   # 33% with 16 time grains
+    )
+
+    for meeting in meetings:
+        duration_time_grains, = rnd.choices(population=counts(duration_distribution),
+                                           weights=weights(duration_distribution))
+        meeting.duration_in_grains = duration_time_grains
+    
+    # Add required attendees using CountDistribution and random.choices
+    required_attendees_distribution = (
+        CountDistribution(count=2, weight=0.36),
+        CountDistribution(count=3, weight=0.08),
+        CountDistribution(count=4, weight=0.02),
+        CountDistribution(count=5, weight=0.08),
+        CountDistribution(count=6, weight=0.10),
+        CountDistribution(count=7, weight=0.05),
+        CountDistribution(count=8, weight=0.05),
+        CountDistribution(count=10, weight=0.05)
+    )
+
+    def add_required_attendees(meeting: Meeting, count: int) -> None:
+        while len(meeting.required_attendances) < count:
+            person = rnd.choice(people)
+            try:
+                meeting.add_required_attendant(person)
+            except ValueError:
+                pass
+
+    for meeting in meetings:
+        count, = rnd.choices(population=counts(required_attendees_distribution),
+                            weights=weights(required_attendees_distribution))
+        add_required_attendees(meeting, count)
+    
+    # Add preferred attendees using CountDistribution and random.choices
+    preferred_attendees_distribution = (
+        CountDistribution(count=1, weight=0.06),
+        CountDistribution(count=2, weight=0.20),
+        CountDistribution(count=3, weight=0.18),
+        CountDistribution(count=4, weight=0.06),
+        CountDistribution(count=5, weight=0.04),
+        CountDistribution(count=6, weight=0.04),
+        CountDistribution(count=7, weight=0.04),
+        CountDistribution(count=8, weight=0.04),
+        CountDistribution(count=9, weight=0.08),
+        CountDistribution(count=10, weight=0.04)
+    )
+
+    def add_preferred_attendees(meeting: Meeting, count: int) -> None:
+        while len(meeting.preferred_attendances) < count:
+            person = rnd.choice(people)
+            try:
+                if not any(ra.person.id == person.id for ra in meeting.required_attendances):
+                    meeting.add_preferred_attendant(person)
+            except ValueError:
+                pass
```

Comment:
Can be simplified using `random.sample`:
```suggestion
        unused_people = list(set(people) - set(meeting.required_attendance))
        sorted_unused_people = sorted(unused_people, key=lambda person: person.id)
        random.sample(sorted_unused_people, count)
```